cmd/deploy/remote.go

```diff
@@ -69,7 +69,8 @@ WORKDIR /okteto/src
 ARG {{ .GitCommitArgName }}
 ARG {{ .InvalidateCacheArgName }}
 
-RUN okteto deploy --log-output=json --server-name="${{ .InternalServerName }}" {{ .DeployFlags }}
+RUN ssh-keyscan {{ join .KnownSshHosts " " }} >> ~/.ssh/known_hosts
```

Comment:
If we are using local ssh key, would it make sense to copy the local known_hosts file instead of generating it from a list we provide (with the implications of it referring to one of the questions you posted in the description like how users would be able to configure it)? cc @pchico83 @jpf-okteto